Sources/PerformanceMetrics/PerformanceMetricsTracker.swift

```diff
@@ -9,6 +9,6 @@
 import Foundation
 
 public protocol PerformanceMetricsTracker: class {
-    func begin(with identifier: PerformanceMetrics.Identifier, metadata: PerformanceMetrics.Metadata)
-    func end(with identifier: PerformanceMetrics.Identifier, metadata: PerformanceMetrics.Metadata)
+    func begin(with identifier: PerformanceMetrics.Identifier)
+    func end(with identifier: PerformanceMetrics.Identifier, metadata: PerformanceMetrics.Metadata?)
```

Comment:
This just hit me now (and it was already like this), but we're tying this protocol to a concrete implementation (`PerformanceMetrics`) 😆 

My suggestion is that we either define those `typealias` inside the protocol, or use the original types 😉

Reply:
Fixed